docs/pages/api-docs/app-bar.json

```diff
@@ -6,16 +6,15 @@
       "type": {
         "name": "enum",
         "description": "'default'<br>&#124;&nbsp;'inherit'<br>&#124;&nbsp;'primary'<br>&#124;&nbsp;'secondary'<br>&#124;&nbsp;'transparent'"
-      },
```

Comment:
Hopefully the changes on https://github.com/mui-org/material-ui/pull/24439/files#r558589220 will help us fix these regressions